homeassistant/components/boschshc/binary_sensor.py

```diff
@@ -0,0 +1,58 @@
+"""Platform for binarysensor integration."""
+import logging
+
+from boschshcpy import SHCSession, SHCShutterContact
+
+from homeassistant.components.binary_sensor import (
+    DEVICE_CLASS_DOOR,
+    DEVICE_CLASS_WINDOW,
+    BinarySensorEntity,
+)
+from homeassistant.const import CONF_IP_ADDRESS
+
+from .const import DOMAIN
+from .entity import SHCEntity
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_entry(hass, config_entry, async_add_entities):
+    """Set up the SHC binary sensor platform."""
+    device = []
```

Comment:
Please rename this to entities.
```suggestion
    entities = []
```